apps/web/app/layout.tsx

```diff
@@ -122,6 +131,9 @@ export default async function RootLayout({ children }: { children: React.ReactNo
             --font-cal: ${calFont.style.fontFamily.replace(/\'/g, "")};
           }
         `}</style>
+        <Script id="i18n-data" strategy="beforeInteractive">
```

Comment:
`beforeInteractive` ensures window.__I18N_DATA__ is set before React hydration of client components